examples/3_Advanced/coil_force_optimization/coil_force_pareto_scans.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python
+
+"""
+coil_force_pareto_scans.py
+--------------------------
+
+This script performs analysis and visualization for stage-two coil optimization results using force metrics. It processes optimization results stored in JSON files, applies engineering and quality filters, computes Pareto fronts for selected objectives, and generates summary plots. The script is intended for use in advanced coil design studies, particularly for quasi-axisymmetric (QA) stellarator configurations.
+
```

Comment:
wasn't this sort of scan done in a recent paper? if I'm not mistaken, can we include a reference to it here?